legal-api/scripts/manual_db_scripts/legal_name_change/transfer_to_new_lear.sql

```diff
@@ -406,11 +406,11 @@ where sq.version != mv.max_version;
 
 
 vset cli.settings.transfer_threads=8
--- aliases -> aliases
-transfer public.aliases from lear_old using
+-- aliases -> alternate_names
+transfer public.alternate_names from lear_old using
 SELECT a.id,
-       a.alias,
-       a.type,
+       a.alias                 as name,
+       a.type                  as name_type,
        a.business_id           as legal_entity_id,
        f.id                    as change_filing_id,
        COALESCE(av.version, 0) as version
```

Comment:
`start_date` in `alternate_names` needs to be populated

Reply:
What kind of value we use for start_date?

Reply:
> `start_date` in `alternate_names` needs to be populated

What about `end_date` in `alterante_names`?

Reply:
> What kind of value we use for start_date?

Think it's `filing.effective_date` for corresponding filing as per my previous conversation with Argus

Reply:
`end_date` is not required.

From what I can see, there are no deletions for `aliases`.  Only updates.

Reply:
Added the `start_date`

Reply:
it needs to be populated in history table too

Reply:
I did it